project/Dependencies.scala

```diff
@@ -0,0 +1,140 @@
+import net.aichler.jupiter.sbt.Import.JupiterKeys
+import sbt.Keys.libraryDependencies
+import sbt.Keys.scalaBinaryVersion
+import sbt._
+
+/**
+ *
+ */
+object BuildType {
+  val Nightly = sys.env.get("TRAVIS_EVENT_TYPE").contains("cron")
+
+}
+object Dependencies {
+
+  object Versions {
+    val Scala211 = "2.11.12"
+    val Scala212 = "2.12.10"
+    val Scala213 = "2.13.1"
+    // Only the root project and the `clusterSharding` project use `ScalaVersions`, all others
+    // use `CustomScalaVersions`
+    val ScalaVersions = Seq(Scala212, Scala213)
+    val CustomScalaVersions = if (BuildType.Nightly) Seq(Scala212, Scala213) else Seq(Scala212, Scala211, Scala213)
```

Comment:
This seems confusing. Why are we building root+clusterSharding with different Scala versions than the rest of the submodules? Can we come up with naming that reflects this?